tests/testHtmIndex.py

```diff
@@ -239,6 +242,16 @@ def testIngest(self):
         cent = make_coord(*tupl)
         cat = loader.loadSkyCircle(cent, self.search_radius, filterName='a')
         self.assertTrue(len(cat) > 0)
+        # test that a catalog can be loaded even with a name not used for ingestion
+        butler = dafPersist.Butler(self.test_repo_path)
+        config = LoadIndexedReferenceObjectsConfig()
+        config.ref_dataset_name = "diff_ref_name"
+        loader = LoadIndexedReferenceObjectsTask(butler=butler, config=config)
+        # This location is known to have objects
+        tupl = (93.0, -90.0)
+        cent = make_coord(*tupl)
```

Comment:
Is there a reason not to strip out the `tupl` variable and just to `make_coord(93.0, -90.0)`?